tests/components/govee_light_local/conftest.py

```diff
@@ -20,6 +21,7 @@ def fixture_mock_govee_api():
     mock_api.turn_on_off = AsyncMock()
```

Comment:
Also, I just thought, I think this fixture can be more efficient:
```py
@pytest.fixture
def mock_airgradient_client() -> Generator[AsyncMock]:
    """Mock an AirGradient client."""
    with (
        patch(
            "homeassistant.components.airgradient.AirGradientClient",
            autospec=True,
        ) as mock_client,
        patch(
            "homeassistant.components.airgradient.config_flow.AirGradientClient",
            new=mock_client,
        ),
    ):
```
This way you don't have to do `with patch(..., new=...)` everytime.

But that can be a separate PR